src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -91,7 +91,7 @@ protected override async Task HandleWebResourceRequest(CoreWebView2WebResourceRe
 					{
 						relativePath = _hostPageRelativePath;
 					}
-					relativePath = Path.Combine(_contentRootDir, relativePath.Replace("/", "\\"));
+					relativePath = Path.Combine(_contentRootDir, relativePath.Replace("/", "\\", StringComparison.Ordinal));
```

Comment:
```suggestion
					relativePath = Path.Combine(_contentRootDir, relativePath.Replace('/', '\\'));
```

Reply:
Thanks , this one is better, and doesn't trigger the code analysis warning.

Let me audit everywhere I changed `string.Replace()` in here.

Reply:
I found about 5 places using 1-letter `string.Replace()`. Should be fixed now.